.github/ISSUE_TEMPLATE.md

```diff
@@ -2,11 +2,7 @@
 ...
 
 #### Technical info
-* Webpack Bundle Analyzer version:
-* Webpack version:
-* Node.js version:
-* npm/yarn version:
-* OS:
+<!-- please include the output of npx envinfo --system --binaries --npmPackages=*webpack* --npmGlobalPackages=webpack-bundle-analyzer -->
```

Comment:
Could we make this a bit easier to copy-paste? :relaxed:

```suggestion

<!--
Please include the output of running the command below:

npx envinfo --system --binaries --npmPackages=*webpack* --npmGlobalPackages=webpack-bundle-analyzer 

-->

<pre>
REPLACE THIS TEXT WITH THE OUTPUT FROM THE COMMAND ABOVE
</pre>
```

This way we would also get the proper formatting, as if users forgot to use code blocks for the output, it would look like this:

  System:
    OS: macOS 10.15.4
    CPU: (8) x64 Intel(R) Core(TM) i7-3740QM CPU @ 2.70GHz
    Memory: 1.38 GB / 16.00 GB
    Shell: 5.7.1 - /usr/local/bin/zsh
  Binaries:
    Node: 12.16.1 - ~/.nvm/versions/node/v12.16.1/bin/node
    Yarn: 1.21.1 - ~/.yarn/bin/yarn
    npm: 6.13.4 - ~/.nvm/versions/node/v12.16.1/bin/npm
  npmPackages:
    assets-webpack-plugin: ^3.9.12 => 3.9.12
    eslint-import-resolver-webpack: ^0.12.1 => 0.12.1
    terser-webpack-plugin: ^2.3.6 => 2.3.6
    webpack: ^4.43.0 => 4.43.0
    webpack-bundle-analyzer: ^3.8.0 => 3.8.0
    webpack-cli: ^3.3.11 => 3.3.11
    webpack-filter-warnings-plugin: ^1.2.1 => 1.2.1


When we'd want it to look like this instead:

<pre>
  System:
    OS: macOS 10.15.4
    CPU: (8) x64 Intel(R) Core(TM) i7-3740QM CPU @ 2.70GHz
    Memory: 1.38 GB / 16.00 GB
    Shell: 5.7.1 - /usr/local/bin/zsh
  Binaries:
    Node: 12.16.1 - ~/.nvm/versions/node/v12.16.1/bin/node
    Yarn: 1.21.1 - ~/.yarn/bin/yarn
    npm: 6.13.4 - ~/.nvm/versions/node/v12.16.1/bin/npm
  npmPackages:
    assets-webpack-plugin: ^3.9.12 => 3.9.12
    eslint-import-resolver-webpack: ^0.12.1 => 0.12.1
    terser-webpack-plugin: ^2.3.6 => 2.3.6
    webpack: ^4.43.0 => 4.43.0
    webpack-bundle-analyzer: ^3.8.0 => 3.8.0
    webpack-cli: ^3.3.11 => 3.3.11
    webpack-filter-warnings-plugin: ^1.2.1 => 1.2.1
</pre>